sled-agent/src/instance.rs

```diff
@@ -355,7 +355,9 @@ impl InstanceInner {
         };
 
         info!(self.log, "Sending ensure request to propolis: {:?}", request);
-        client.instance_ensure().body(request).send().await?;
+        let result = client.instance_ensure().body(request).send().await;
+        info!(self.log, "result of instance_ensure call is {:?}", result);
+        result?;
 
         // Monitor propolis for state changes in the background.
         let monitor_task = Some(tokio::task::spawn(async move {
```

Comment:
Since they both use no-timeout clients now mind just changing this to pass the same client to `monitor_task`?

Reply:
(James is on paternity leave, working with alan to land outstanding PRs.)